src/codeu/chat/client/core/ConversationContext.java

```diff
@@ -102,4 +102,8 @@ public MessageContext findMessageByUuid(Uuid id) {
     return null;
   }
   
+  public void changeAuthority(User targetUser, String authorityParameter)
```

Comment:
Oh yeah, and targetUser should be a Uuid! If all these Uuids are confusing you can write nested classes that will clear things up nicely.